src/MenuMixin.js

```diff
@@ -125,14 +120,6 @@ const MenuMixin = {
       e.preventDefault();
       updateActiveKey(this.getStore(), this.getEventKey(), activeItem.props.eventKey);
 
-      if (typeof callback === 'function') {
-        callback(activeItem);
-      }
-
-      return 1;
-    } else if (activeItem === undefined) {
```

Comment:
activeItem is either `null` or an object, never hit `undefined`. `null` case will be taken care by caller.